sparkmagic/sparkmagic/tests/test_configuration.py

```diff
@@ -6,7 +6,6 @@
 from sparkmagic.livyclientlib.exceptions import BadUserConfigurationException
 from sparkmagic.utils.constants import AUTH_BASIC, NO_AUTH
 
-
```

Comment:
I think that there was a lint problem since this file had no changes except for this space, if possible preserve these spaces, so we can a have a contribution later to address any linter issues